README.md

```diff
@@ -1,2 +1,159 @@
 # time-monkey
 Base application for management of employees working hours
+
+## Environment set-up
+
+### Ubuntu
+
+1. Install Python 3.7.0 by running in Terminal `sudo apt install python3.7`. You can verify the installation by running: 
+    `python3.7 --version`
+
+2. Install PIP with `sudo apt install python3-pip`.  You can verify the installation by running: `python3.7 -m pip`.
```

Comment:
Same situation here. Please move `You can verify....` to new line. Same with command `python3.7 -m pip`

Reply:
Done